devolving_music/lib/song_scores.py

```diff
@@ -110,42 +132,86 @@ def get_quality_list(self,
 
         return final_quality_list
 
-    ###
-    # static methods can be moved into helper function
-    ###
     @staticmethod
     # sorts in ascending order
     def get_info_sort(
-            score_suite_list: List["ScoreSuite"]) -> List["ScoreSuite"]:
+            score_suite_list: List["ScoreSuite"],
+            get_informed=False,
+            informed_threshold=1) -> List["ScoreSuite"]:
         """
-        Returns a list of song suites ordered by low information songs with the lowest information score 
-        are randomly shuffled before being added to the list
-        
+        Returns a list of song suites ordered by low information songs with the lowest information score
+        randomly shuffled before being added to the list
+
         """
-        score_suite_list.sort(key=lambda sub: sub.info_score)
+        score_suite_list = sorted(score_suite_list, key=lambda sub: sub.info_score)
         rightend = 0
-        init = score_suite_list[0].info_score
+        if (get_informed):
+            init = informed_threshold
+        else:
+            init = score_suite_list[0].info_score
```

Comment:
```suggestion
        if not get_informed:
            informed_threshold = score_suite_list[0].info_score
```

Why not just keep using `informed_threshold`? It's a good name.